src/catalog/Indexer/AttributesIndexer.php

```
@@ -3,7 +3,6 @@
 namespace StreamX\ConnectorCatalog\Indexer;
 
 use Psr\Log\LoggerInterface;
-use Streamx\Clients\Ingestion\Exceptions\StreamxClientException;
```

All StreamX related code is now in a single class - connector's StreamxClient class
 - remove references to Streamx PHP Client code from other places. The above exception was not handled anyway